### ISSUE TITLE
modular views

### ISSUE THREAD

**Issue by abdulhannanali** [roonyh/fossasia-gci-companion#10]
the view is a one really large file the way it's written,
how about separating the views into multiple files in order to improve code modularity, 
also how about using the Jade

**Comment by abdulhannanali** [comment]
Jade for views. jade-lang.com

**Comment by roonyh** [comment]
yes we need modular views and some templating tool. not sure about jade though. again lets wait until we have a clear folder structure. I will get to this after #2 and #3 are done
